content/en/users/datatransfer/clients/_index.md

```diff
@@ -0,0 +1,315 @@
+---
+title: "Clients"
+linkTitle: "Clients"
+type: docs
+description: "Documentation related to EGI Data Transfer Clients"
+weight: 30
+---
+
+## Overview 
+
+The FTS3 service offers command line clients both on C++ and Python. As the CLI in C++ is deprecated we are going to focus this section on the python CLI.
```

Comment:
```suggestion
The FTS3 service offers command line clients both in C++ and Python. As the CLI in C++ is deprecated we are going to focus this section on the python CLI.
```

Reply:
Not sure user cares about the fact that a CLI is C++ or Python, furthermore if one of them is deprecated/not documented